src/extension/tests/TestExtEnvHandler.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 #
```

Comment:
All-up -- ensure tests mock running on both older agents without telemetry and those with.